src/BaseTemplates/StarterWeb/project.json

```diff
@@ -71,6 +71,7 @@ $endif$
     "include": [
       "wwwroot",
       "Views",
+      "Areas\**\Views",
```

Comment:
Does this do the right thing? Just making sure you double checked that this is actually the right include